### ISSUE TITLE
Combining grapheme joiner (CGJ) with double diacritics

### ISSUE THREAD

**Issue by moyogo** [notofonts/latin-greek-cyrillic#226]
u͡͏́i (0075 0361 034F 0301 0069) should have the combining acute 0301 on top of the double inverted breve 0361 when the CGF 034F is between the two, whereas ú͡i (0075 0361 0301 0069) should be rendered as the canonically equivalent ú͡i (0075 0301 0361 0069).
See http://www.unicode.org/versions/Unicode8.0.0/ch07.pdf#page=43

Another example is t͡͏̇s (0074 0361 034F 0307 0073) which should be used to transliterate the Cyrillic ҵ in ALA-LC romanization in many library catalogues.
Note: it is often encoded with the half marks from MARC, for example "Akhȧ︠t︡sarashẇa" in http://catalog.nypl.org/search/o38741987, or without the CGJ, for example "Akhaṫ͡sarashẇa" in https://catalog.lib.uchicago.edu/vufind/Record/2778324.

**Comment by simoncozens** [comment]
This is fixed in Serif but not in Sans, and I'm not sure why not; breveinverteddoublecomb has a top anchor, and dotaccentcomb (obviously) has a _top anchor, but they're not attaching.

```
$ shape fonts/NotoSerif/googlefonts/variable-ttf/NotoSerif\[wdth,wght\].ttf 't͡͏̇s'
[t=0+352|uni0361=0@125,79+0|space=0+0|uni0307=0@125,368+0|s=4+451]
```
![shape](https://user-images.githubusercontent.com/106728/213460814-5c1211e9-a18a-44ec-98a1-0e4db4789af7.png)

```
$ shape fonts/NotoSans/googlefonts/variable-ttf/NotoSans\[wdth,wght\].ttf 't͡͏̇s'
[t=0+361|uni0361=0@131,51+0|space=0+0|uni0307=0+0|s=4+479]
```
![shape](https://user-images.githubusercontent.com/106728/213460902-5b380e0e-d314-4d19-8737-31b9099d7a07.png)

**Comment by simoncozens** [comment]
More debugging, Noto Serif has a GDEF entry for uni034F (class 3, of course) but Noto Sans does not, so it's probably that. Indeed, Noto Sans doesn't seem to have ClassDefs in the GDEF table at all! This turns out to be a weird tooling issue: when I compile Noto Sans from individual UFOs, the GDEF table is correct, but when it's compiled from the designspace file, GDEF is not added to the generated (debug) fea file...

OK, so in the designspace lib, there is an explicit com.github.googlei18n.ufo2ft.featureWriters entry, with KernFeatureWriter and MarkFeatureWriter (set to mode=skip?!) but no GDEFFeatureWriter. (But not with Noto Serif?) Aaah, it's because we have our own manual kern rules, and so googlefonts/glyphsLib#764 applies.

**Comment by simoncozens** [comment]
This'll be fixed when googlefonts/glyphsLib#849 lands.

**Comment by simoncozens** [comment]
It is indeed fixed now.
